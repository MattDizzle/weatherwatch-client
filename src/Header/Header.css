header {
    background-color: white;
}

.logo {
    width: 10vw;
}