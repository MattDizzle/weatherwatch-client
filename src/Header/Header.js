import React, { Component } from "react";
import {CurrentContext} from "../CurrentContext";
import logo from "../images/weather-watch-logo.png";
import "./Header.css";

export default class Header extends Component {
  static contextType = CurrentContext;
  render() {
    return (
      <header className='animate__animated animate__slideInDown'>
     <a href='https://weatherwatch.club/'><img src={logo} alt="Logo" className="logo" /></a> 
      </header>
    );
  }
}
