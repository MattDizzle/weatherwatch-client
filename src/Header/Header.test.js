
import React from "react";
import ReactDOM from "react-dom";
import renderer from 'react-test-renderer';
import Header from "./Header";

describe(`Header Component`, () => {
    describe(`Smoke test`, () => {
        it("should render without crashing", () => {
            const div = document.createElement('div');
            ReactDOM.render(<Header/>, div);
            ReactDOM.unmountComponentAtNode(div);
          });
    })

    describe(`Snapshot test`, () => {
        it(`Renders the UI as expected`, () => {
          const div = document.createElement('div');
            const tree = renderer.create(<Header/>).toJSON()
            expect(tree).toMatchSnapshot()
        });
    });

})


