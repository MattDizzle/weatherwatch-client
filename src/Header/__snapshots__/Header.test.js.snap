// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Snapshot test Renders the UI as expected 1`] = `
<header>
  <img
    alt="Logo"
    className="logo"
    src="weather-watch-logo.png"
  />
</header>
`;
